src/pages/Kavita/main.ts

```diff
@@ -0,0 +1,200 @@
+      if (provider === 'ANILIST' && seriesDetails.anilistLink) return seriesDetails.anilistLink;
+      if (provider === 'KITSU' && seriesDetails.kitsuLink) return seriesDetails.kitsuLink;
+      return false;
+    },
+    readerConfig: [
+      {
+        current: {
+          callback: () => chapterDetails.currentPageNum as any,
+          mode: 'callback',
+        },
+        total: {
+          callback: () => chapterDetails.totalPageNum as any,
+          mode: 'callback',
+        },
+      },
+    ],
+  },
+  overview: {
+    getTitle() {
+      if (!seriesDetails.seriesTitle) throw 'No name';
+      return seriesDetails.seriesTitle;
+    },
+    getIdentifier(url) {
+      if (!seriesDetails.seriesID) throw 'No series id';
+      return seriesDetails.seriesID;
+    },
+    uiSelector(selector) {
+      return j.$('h4').first().after(j.html(selector));
+    },
+  },
+  init(page) {
+    api.storage.addStyle(
+      require('!to-string-loader!css-loader!less-loader!./style.less').toString(),
+    );
+
+    let progressUpdater;
+    utils.changeDetect(
+      () => {
+        check();
+      },
+      () => {
+        return utils.urlStrip(window.location.href);
+      },
+    );
+    waitForPageLoad(check);
+
+    async function getSeriesDetails() {
+      const seriesID = window.location.pathname.split('/')[4];
+
+      const url = `${window.location.origin}/api/series/${seriesID}`;
+      const response = await apiCall(url);
+      const responseJSON = JSON.parse(response.responseText);
+      con.log('Series Details Retrieved', responseJSON);
+
+      seriesDetails.seriesTitle = responseJSON.name;
+      seriesDetails.seriesID = seriesID;
+      seriesDetails.totalPageNum = responseJSON.pages;
+    }
+
+    async function getChapterDetails() {
+      const chapterID = window.location.pathname.split('/')[6];
+
+      const url = `${window.location.origin}/api/reader/chapter-info?chapterid=${chapterID}`;
+      const response = await apiCall(url);
+      const responseJSON = JSON.parse(response.responseText);
+      con.log('Chapter Info Retrieved', response);
+
+      chapterDetails.volumeNum = responseJSON.volumeNumber;
+      chapterDetails.chapterNum = responseJSON.chapterNumber < 0 ? 0 : responseJSON.chapterNumber;
+      chapterDetails.totalPageNum = responseJSON.pages;
+    }
+
+    function waitForPageLoad(callback) {
+      utils.waitUntilTrue(
+        () => j.$('.spinner-border span').length,
+        () => {
+          utils.waitUntilTrue(
+            () => j.$('.spinner-border span').length === 0,
+            () => {
+              callback();
+            },
+          );
+        },
+      );
+    }
+
+    function getLinks() {
+      seriesDetails.anilistLink =
+        j.$('[href^="https://anilist.co/manga/"]').attr('href') ?? seriesDetails.anilistLink;
+      seriesDetails.malLink =
+        j.$('[href^="https://myanimlist.net/manga/"]').attr('href') ?? seriesDetails.malLink;
```

Comment:
I'm not kavita user, but isn't that as typo? `https://myanimlist.net/manga/` -> `https://myanimelist.net/manga/`